doc/generate_cet.py

```diff
@@ -3,7 +3,144 @@
 import pathlib
 
 
+def provide_ignored():
+    """Returns lists of ignored attributes and fields."""
+    ignored_attributes = ["EFDB_Name", "Description"]
+    ignored_fields = ["Description", "Alias", "Device", "Property", "Action", "Value", "Subsystem", "Version", "Author", "Explanation"]
+    return ignored_attributes, ignored_fields
+
+def add_generics(cf, subsystem, set_name, has_generics, has_specific):
+    """Adds generic topics to the rst file in the appropriate section.
+
+    Parameters
+    ----------
+    cf : `IO`
+        A file object for writting to each CSC rst file.
+    subsystem : `str`
+        The name of the CSC.
+    set_name : `str`
+        The name of the topic set.
+    has_generics : `bool` or `List` of `str`
+        It either has generic topics(yes) or not(no) or a list of supported topics.
+    has_specific : `List` of `str`
+        A list for specifying whether a CSC has specific kinds of topics or not(command, event, telemetry)
```

Comment:
What is in the list? Three booleans? The names of the kinds? Does case matter? I am guessing it is a list of one or more of "command", "event", "telemetry" but I honestly don't know.